### DISCUSSION TITLE
blujay

### DISCUSSION THREAD

**Discussion by Raytwo** [Raytwo/ARCropolis#387] [Announcements]
### Bugfixe(s)
* Fix shared files not working in config.json (@blu-dev).

# Read this
Skyline is not bundled with ARCropolis anymore.
Please read [this issue](https://github.com/Raytwo/ARCropolis/issues/157) to know how to proceed from now on.

<hr /><em>This discussion was created from the release <a href='https://github.com/Raytwo/ARCropolis/releases/tag/v3.5.1'>blujay</a>.</em>

**Comment by TK-Wolf** [comment]
I downloaded this via the in-game prompt, after being on the previous version, but now smash crashes on launch. Any idea how to diagnose or resolve this?

**Comment by Naughtypoked** [comment]
This update has seemingly broken something to do with file addition. Many of previously working skins no longer work. No syntax changes or errors in config.json. A good example is "Great Ape Vegeta" by Mastaklo or "Buff Kirby" by JJ314.

**Reply by Naughtypoked** [reply]
Just wanted and follow up with a note that reverting back to prior builds of Arcropolis seems to have resolved the issue. Few others are reporting the same with other skin mods not loading textures/ materials that use file addition (however work in previous builds and no syntax issue can be found in their config.json files for skins). I hope this helps in diagnosing the issue. Thanks for developing these great tools and sharing :)
